### ISSUE TITLE
How to select checkpoint when fine-tuning on MSRVTT

### ISSUE THREAD

**Issue by chaochen99** [FingerRec/OA-Transformer#9]
Dear Sir,

During fine-tuning of MSRVTT, we found no validation set in the code. How to choose a checkpoint?

Looking forward to your reply.

Thanks！

**Comment by FingerRec** [comment]
Actually, we select the checkpoint with best zero-shot result in MSRVTT.
